src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -187,6 +187,17 @@ def load_arguments(self, _):
     with self.argument_context('aks upgrade') as c:
         c.argument('kubernetes_version', completer=get_k8s_upgrades_completion_list)
 
+    with self.argument_context('aks upgrade-connector') as c:
+        c.argument('aci_resource_group', help='The resource group to create the ACI container groups')
```

Comment:
You can omit these `help=` arguments, since the `short-summary:` text you have in _help.py will be used (and that's the preferred location for such text).